docs/content/latest/explore/observability/prometheus-integration/linux.md

```diff
@@ -211,7 +224,9 @@ avg(irate(rpc_latency_count{server_type="yb_cqlserver", service_type="SQLProcess
 Optionally, you can shut down the local cluster created in Step 1.
 
 ```sh
-$ ./bin/yb-ctl destroy
+$ ./bin/yugabyted destroy --base_dir=node-1/ && \
```

Comment:
same as the other one(s): let's standardize the destroy commands everywhere in explore section